content/en/docs/ambient/install/helm-installation/index.md

```diff
@@ -28,7 +28,21 @@ we encourage the use of Helm to install Istio for use in ambient mode. Helm help
 
 *See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation.*
 
-## Install the components
+{{< gloss "ambient" >}}Ambient{{< /gloss >}} is made up of multiple components - for production deployments, it is strongly recommended to install the components separately to allow for more control during upgrades. Consult the [operations guidelines](/docs/ops/deployment) for production deployment considerations.
+
+If you just want to try out Istio ambient in a non-production cluster, to keep things simple it is recommended to use the `ambient` Helm wrapper chart which installs the minimum set of required components together.
+
+## Install the components together
+
+The `ambient` Helm wrapper chart composes all the components that make up an ambient installation into one Helm chart:
```

Comment:
is "wrapper chart" a term of art?  I don't see it in the Helm docs

Reply:
very much not.

This is a relatively common pattern (used in Argo, etc) that doesn't really have a specific Helm definition.

We could say "parent chart" or "composite chart" but to Helm it's just a Helm chart with Helm dependencies (and nothing else)

Reply:
I've just removed `wrapper` as a term, and talked around the informal term. Open to better ideas if anyone has them.